tests/regression/66-longjmp/18-simple-else.c

```diff
@@ -0,0 +1,39 @@
+// PARAM: --enable ana.int.interval --enable ana.int.enums
+#include <stdio.h>
+#include <stdlib.h>
+#include <setjmp.h>
+
+jmp_buf env_buffer;
+int global = 0;
+
+int fun() {
+   global = 2;
+   longjmp(env_buffer, 2);
+}
+
+
+int main () {
+   int val;
+   jmp_buf env_buffer2;
+
+   __goblint_check(global == 0);
+
+   /* save calling environment for longjmp */
+   val = setjmp( env_buffer );
+
+   __goblint_check(global == 0); //UNKNOWN!
+   __goblint_check(global == 2); //UNKNOWN!
+   __goblint_check(global == 8); //FAIL!
+
+   if( val != 0 ) {
+      printf("Returned from a longjmp() with value = %i\n", val);
+      __goblint_check(val == 2);
+      __goblint_check(global == 2); //TODO (requires path-sensitivity distinguishing between returns)
```

Comment:
> Seems like this would be fairly easy to achieve, given that setjmp returns at most twice

Actually, it may return more than twice. But yes, there should be no explosion of paths because of it.

Reply:
I added a test that does this!